api/runner/async_runner.go

```diff
@@ -52,8 +51,7 @@ func getCfg(task *models.Task) *Config {
 		Timeout: time.Duration(*task.Timeout) * time.Second,
 		ID:      task.ID,
 		AppName: task.AppName,
-		Stdout:  stderr,
-		Stderr:  stderr,
+		Stdout:  ioutil.Discard,
```

Comment:
> What is the benefit of logging the async result?

So you can actually inspect what an async is doing. Or putting differently, this is a subtle change that I wouldn't mind as long as we make clear that this is the intention.